api/lib/fiatvalues-api.js

```diff
@@ -76,7 +78,7 @@ class fiatValueAPI {
     this.fiatValuesAPIDictionary = denomToCoinGeckoIDDictionary
     this.fiatValuesAPIReverseDictionary = geckoIDToDenomDictionary
     this.fiatCurrenciesSymbolsDictionary = fiatCurrenciesSymbolsDictionary
-    this.coins = ['ATOM', 'LUNA', 'KRT', 'SDT', 'KAVA', 'BNB', 'KSM'] // Lunie coins currently being traded in the open
+    this.coins = ['ATOM', 'LUNA', 'KRT', 'SDT', 'KAVA', 'BNB', 'KSM', 'DOT'] // Lunie coins currently being traded in the open
```

Comment:
can we handle this somehow without needing to hardcode?

Reply:
Yes, I think the best would be to have this stored in DB too

Reply:
> without needing to hardcode

Reply:
:thinking: any suggestions?

Reply:
try to get the coins and if they are not on coingecko ignore them

Reply:
:shrug: maybe we just leave it like this (although I prefer the DB-centered solution)

Reply:
let's leave as is